src/components/common/badge/EmojiBadge.jsx

```diff
@@ -0,0 +1,31 @@
+//이모지 뱃지 컴포넌트
+
+import { cva } from 'class-variance-authority';
+import PropTypes from 'prop-types';
+import { cn } from '@/utils/style';
+const emojiBadgeStyle = cva(
+  'inline-flex items-center justify-center rounded-full bg-black/50 text-white font-16-regular',
+  {
+    variants: {
+      size: {
+        s: 'px-3 py-2 h-[28px] font-14-regular',
+        md: 'px-3 py-2 h-[36px]',
+        lg: 'px-4 py-2 h-[38px]',
+      },
+    },
+  }
+);
+const EmojiBadge = ({ emoji, count, size = 'md' }) => {
+  return (
+    <span className={cn(emojiBadgeStyle({ size }))}>
+      <span className="mr-1.5">{emoji}</span>
+      {count !== undefined && <span>{count}</span>}
+    </span>
+  );
+};
+EmojiBadge.propTypes = {
+  emoji: PropTypes.string.isRequired,
+  count: PropTypes.number,
+  size: PropTypes.oneOf(['s', 'md', 'lg']),
+};
+export default EmojiBadge;
```

Comment:
코드 가독성을 위해 각 구문별로 구분되도록 줄바꿈을 넣어주면 좋을 것 같습니다!
```suggestion
import { cva } from 'class-variance-authority';
import PropTypes from 'prop-types';
import { cn } from '@/utils/style';

const emojiBadgeStyle = cva(
  'inline-flex items-center justify-center rounded-full bg-black/50 text-white font-16-regular',
  {
    variants: {
      size: {
        s: 'px-3 py-2 h-[28px] font-14-regular',
        md: 'px-3 py-2 h-[36px]',
        lg: 'px-4 py-2 h-[38px]',
      },
    },
  }
);

const EmojiBadge = ({ emoji, count, size = 'md' }) => {
  return (
    <span className={cn(emojiBadgeStyle({ size }))}>
      <span className="mr-1.5">{emoji}</span>
      {count !== undefined && <span>{count}</span>}Expand commentComment on line R22Code has comments. Press enter to view.
    </span>
  );
};

EmojiBadge.propTypes = {
  emoji: PropTypes.string.isRequired,
  count: PropTypes.number,
  size: PropTypes.oneOf(['s', 'md', 'lg']),
};

export default EmojiBadge;
```